components/Common/CodeBox/CodeTabs/index.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { FC } from 'react';
+
+import { CodeBox } from '@/components/Common/CodeBox';
+import { CodeTabs } from '@/components/Common/CodeBox/CodeTabs';
+
+type Story = StoryObj<typeof CodeTabs>;
+type Meta = MetaObj<typeof CodeTabs>;
+
+const TabsContent: FC = () => (
+  <>
+    <TabsPrimitive.Content key="mjs" value="mjs">
+      <CodeBox language="JavaScript (MJS)">
+        <code>
+          {`import * as http from 'http';
```

Comment:
Can you extract this to a constant? So that we don't mix the text with the code?